sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/Utility.java

```diff
@@ -318,7 +318,7 @@ public static OffsetDateTime parseDate(String dateString) {
                         }
                     }
 
-                    return ByteBuffer.wrap(cache);
+                    return ByteBuffer.wrap(cache, 0, offset);
```

Comment:
Caught a bug where "blockSize" in stream->flux converter was greater than last piece of data in the stream.